llama_github/rag_processing/rag_processor.py

```diff
@@ -290,173 +291,182 @@ def _split_content_into_chunks(self, content: str, language: Optional[str] = Non
         chunks = splitter.split_text(content)
         return chunks
 
-    def _arrange_issue_search_result(self, issue_search_result: dict) -> dict:
+    def _arrange_issue_search_result(self, issue_search_result: dict) -> List[Dict[str, str]]:
         """
-        arrange the result of issue search
+        Arrange the result of Issue search with metadata.
 
         Args:
-            issue_search_result (dict): the result of issue search.
+            _arrange_issue_search_result (dict): The result of Issue search.
 
         Returns:
-            dict: the arranged result of issue search.
+            List[Dict[str, str]]: The arranged result of Issue search with metadata.
         """
         arranged_results = []
 
         for result in issue_search_result:
             content = result['issue_content']
+            url = result.get('url', '')
```

Comment:
The `url` parameter from the search issue result list comes from the GitHub Search Issue API, which provides an issue API URL (e.g., `https://api.github.com/repos/quarkusio/quarkus/issues/1957`). This is not the HTML URL. We could consider transforming it to the official GitHub issue webpage URL (e.g., `https://github.com/quarkusio/quarkus/issues/1957`).

Please kindly update the code to reflect this change.